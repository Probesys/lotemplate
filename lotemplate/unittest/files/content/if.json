{
  "tutu": {
    "type": "text",
    "value": "oui"
  },
  "tata": {
		"type": "text",
		"value": "foo"
  },
  "my_func(toto)": {
		"type": "text",
		"value": "my_func_value"
  },
  "odoo(status)": {
    "type": "text",
    "value": "SARL"
  }
}