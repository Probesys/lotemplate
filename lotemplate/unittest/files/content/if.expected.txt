
Tests of if statements

We check the values with the tutu = oui


This paragraph should be displayed (==oui)







This paragraph should be displayed (!= glouglou)

several lines


the foo variable

La fonction a bien été prise en compte

End of the test
